src/set_theory/game/nim.lean

```diff
@@ -248,4 +258,97 @@ lemma equiv_nim_iff_grundy_value_eq {G : pgame.{u}} (hG : impartial G) (O : ordi
 lemma nim.grundy_value (O : ordinal.{u}) : grundy_value (nim O) = O :=
 by rw ←equiv_nim_iff_grundy_value_eq
 
+lemma equiv_iff_grundy_value_eq (G H : pgame) [G.impartial] [H.impartial] :
+  G ≈ H ↔ grundy_value G = grundy_value H :=
+(equiv_congr_left.1 (equiv_nim_grundy_value H) _).trans $ equiv_nim_iff_grundy_value_eq _ _
+
+lemma grundy_value_zero : grundy_value 0 = 0 :=
+by rw [(equiv_iff_grundy_value_eq 0 (nim 0)).1 (equiv_symm nim.zero_first_loses), nim.grundy_value]
+
+lemma equiv_zero_iff_grundy_value {G : pgame.{u}} (hG : impartial G) : G ≈ 0 ↔ grundy_value G = 0 :=
+by rw [equiv_iff_grundy_value_eq, grundy_value_zero]
+
+lemma grundy_value_nim_add_nim (n m : ℕ) : grundy_value (nim n + nim m) = nat.lxor n m :=
+begin
+  induction n using nat.strong_induction_on with n hn generalizing m,
+  induction m using nat.strong_induction_on with m hm,
+  rw [grundy_value_def],
+
+  -- We want to show that `n xor m` is the smallest unreachable grundy value. We will do this in two
```

Comment:
```suggestion
  -- We want to show that `n xor m` is the smallest unreachable Grundy value. We will do this in two
```
Here and below, should "Grundy" be capitalized when talking about the mathematical concept (as opposed to the Lean name)?